content/intro-to-storybook/react/en/creating-addons.md

```diff
@@ -38,81 +35,62 @@ export default {
 };
 ```
 
-<!-- -->
-
 ## Setup
 
-We've outlined what our addon will do, time to setup our local development environment. We need some additional packages in our project. More specifically:
-
-<!-- it would be nice that the readme files would have some minimal information for each package-->
-
-- 📦 [@storybook/api](https://www.npmjs.com/package/@storybook/api) for Storybook API usage.
-- 📦 [@storybook/components](https://www.npmjs.com/package/@storybook/components) to use Storybook's UI components.
-- 📦 [@storybook/theming ](https://www.npmjs.com/package/@storybook/theming) for styling.
-- 🛠 [@babel/preset-react](https://babeljs.io/docs/en/babel-preset-react) to transpile correctly some of React's new features.
-
-Open a console, navigate to your project folder and run the following command:
-
-<!--using npm here until the whole tutorial set is moved into npm or yarn issue #153-->
-
-```bash
-  yarn add --dev @storybook/api @storybook/components @storybook/theming @babel/preset-react
-```
+We've outlined what our addon will do, it's time to start working on it.
 
-We'll need to make a small change to the `.babelrc` file we created earlier. We need to add a reference to the `@babel/preset-react` package.
+Inside your `.storybook` folder, create a new one called `addons` and inside it a new file called `register.js`.
 
-Your updated file should look like this:
+And that's it! Simple isn't it?
 
-```json
-{
-  "presets": ["@babel/preset-react"],
-  "plugins": ["macros"]
-}
-```
+<div class="aside">We're going to use the<code>.storybook</code> folder as a placeholder for our addon. The reason behind this, is to maintain a straightforward approach and avoid complicating it too much. Should this addon be transformed into a actual addon it would be best to move it to a separate package with it's own file and folder structure.</div>
 
 ## Writing the addon
 
-We have what we need, it's time to start working on the actual addon.
-
-Inside the `.storybook` folder create a new folder called `addons` and inside, a file called `design-assets.js` with the following:
+Add the following to your recently created file:
 
 ```javascript
-//.storybook/addons/design-assets.js
-import React from "react";
-import { AddonPanel } from "@storybook/components";
-import { addons, types } from "@storybook/addons";
-
-addons.register("my/design-assets", () => {
-  addons.add("design-assets/panel", {
-    title: "assets",
+//.storybook/addons/register.js
```

Comment:
Should we call this file `design-addon/register.js` maybe? (I assume it needs to be called `.../register.js`)

Reply:
@tmeasday sounds a bit more clearer to what's intended with the section. I've updated the snippets to reflect this with the latest commit. I'm going to mark it as resolved.